### DISCUSSION TITLE
Unfortunately we weren't able to approve your request!

### DISCUSSION THREAD

**Discussion by Huzaifa1910** [community/community#52027] [GitHub Education]
### Select Topic Area

Question

### Body

I'm uploading my original id card for the approval of document verification for GitHub developer student package, my uni doesn't provide any portal so i don't have any other proof except that ID card but its not approving, any idea?

**Comment by Anmol-Baranwal** [comment]
If you have submitted your original ID card for document verification for the GitHub Developer Student Pack and it is not being approved, you can try the following steps:

1. Make sure that the ID card you have submitted is clear and legible. Ensure that the information on the card is visible and can be read easily. If necessary, take a high-quality photo of the ID card with good lighting.

2. Double-check that the information on the ID card matches the information you have provided on the verification form. The name, date of birth, and other personal information should match the information you have submitted.

**Reply by Huzaifa1910** [reply]
I registered myself in 2022 and now its asking me to reverify my academic affiliation and not accepting my card

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Huzaifa1910 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Huzaifa1910** [reply]
Thanks @queenofcorgis
